libs/core/kiln_ai/adapters/extractors/registry.py

```
@@ -1,15 +1,22 @@
+from typing import Callable
+
+from google import genai
```

Pylint (3.3.7)

[error] No name 'genai' in module 'google'

(E0611)